src/custom_init.cr

```diff
@@ -0,0 +1,32 @@
+class LuckyCli::CustomInit < LuckyCli::Init
+  def run
+    project_name = ProjectName.new(ARGV[1]? || "")
```

Comment:
Sanitization, validation, and validation messages are now in the `ProjectName` object